@kiva/kv-components/vue/KvRadio.vue

```diff
@@ -151,7 +151,7 @@ export default {
 			modelValue,
 		} = toRefs(props);
 
-		let uuid = ref(`kvr-${nanoid(10)}`);
+		const uuid = ref(null);
```

Comment:
i think we want to keep the setup init of the uuid as well. Making it a const should be fine though.